3.5.0/scala2.12-java11-python3-r-ubuntu/Dockerfile

```diff
@@ -20,7 +20,10 @@ USER root
 
 RUN set -ex; \
     apt-get update; \
-    apt-get install -y python3 python3-pip; \
+    apt install -y software-properties-common; \
+    add-apt-repository ppa:deadsnakes/ppa; \
+    apt install python3.10; \
```

Comment:
Is there any special reason why we use the python 3.10? I prefer to use os default python3 version from matainence cost view.

Reply:
Rolled this back. My initial idea was to propose adding images with multiple python versions to the repo (java17-python3.10, java17-python3.9, etc), but now that I think about it - probably not a lot of community members would benefit from this and it would clutter up the repository quite a bit.

And those people who need to have specific python versions (like me) can just take a base image and install whatever python version they want.